src/status_im/common/home/banner/style.cljs

```diff
@@ -61,7 +62,7 @@
    {:transform [{:translate-y (animated-card-translation-y scroll-shared-value)}]}
    {:z-index          3
     :position         :absolute
-    :top              (+ (safe-area/get-top) 192)
+    :top              (+ (safe-area/get-top) 192 -8)
```

Comment:
Could explain why we moved the `8` to be a `-8` here?

Also should this be a constant too?

Reply:
A leftover, it should have been subtracted to the `192`.And the reason is that we added a spacing of 8 that never existed on designs, this fix properly aligns the navbar as expected.